identity/extension_credentials.go

```diff
@@ -35,6 +35,7 @@ func (r *SchemaExtensionCredentials) Run(_ jsonschema.ValidationContext, s schem
 			}
 		}
 
+		// TODO--lowercasing something (?jsonnet? - ?no idea?)
 		r.v = stringslice.Unique(append(r.v, strings.ToLower(fmt.Sprintf("%s", value))))
 		cred.Identifiers = r.v
```

Comment:
For me it looks like credentials parsed with jsonnet, but not sure

Reply:
Yeah. It looks like we would need to change it. It is part of the JSON schema validation.

Reply:
It registers an extension that checks if identifiers are unique when having multiple schemas applied.